cliquet/__init__.py

```diff
@@ -26,7 +26,7 @@
 DEFAULT_SETTINGS = {
     'cliquet.backoff': None,
     'cliquet.batch_max_requests': 25,
-    'cliquet.cache_backend': 'cliquet.cache.redis',
+    'cliquet.cache_backend': 'cliquet.cache.memory',
```

Comment:
It is why I created the PR in the first place and it is the only thing left that this PR doesn't handle (having different default for kinto and cliquet on some settings.)